src/cache.ts

```diff
@@ -102,6 +109,14 @@ export default class Cache {
         let value;
         if (entry) {
             value = entry.value;
+            if (this.stdTTL > 0) {
+                const deadline = entry.created.getTime() + this.stdTTL * 1000;
+                const now = (new Date()).getTime();
+                if (deadline < now) {
+                    await this.remove(key);
```

Comment:
Wouldn't it be better to not wait for removal? If it's a cache miss, we want to go to the origin right away.